bin/varnishd/waiter/cache_waiter_epoll.c

```diff
@@ -30,6 +30,8 @@
  * - thank you, Marc Alexander Lehmann
  */
 
+#include <stdlib.h>
```

Comment:
Should move after `config.h`, did you put it here because it would otherwise fail to build?

Reply:
fixed and force pushed